updatecli/updatecli.d/maven.yml

```diff
@@ -13,30 +13,30 @@ scms:
       branch: "{{ .github.branch }}"
 
 sources:
-  mavenVersion:
-    kind: githubRelease
-    name: Get the latest Maven version
+  # Retrieving Maven from packer-images to synchronize its version across our infra
+  # See https://github.com/jenkins-infra/docker-inbound-agents/issues/18
+  getMavenVersionFromPackerImages:
+    kind: file
+    name: Get the latest Maven version set in packer-images
     spec:
-      owner: "apache"
-      repository: "maven"
-      token: "{{ requiredEnv .github.token }}"
-      username: "{{ .github.username }}"
-      versionFilter:
-        kind: latest
+      file: https://raw.githubusercontent.com/jenkins-infra/packer-images/main/provisioning/tools-versions.yml
```

Comment:
Using the `main` branch means that we can have an opened PR here about a merged PR not released yet.
It means that we can have a difference until it is released.

Not really a problem at first sight, but better to put a comment here.

Otherwise, we could merge this manifest into the "agent" manifest, where we get the version of the packer image, so the PR will change syncrhonously the AMI ids, the Azure VM version AND the maven version.


WDYT?

Reply:
Let discuss this together on Monday :)